<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockRxOpt</class>
 <widget class="QDockWidget" name="DockRxOpt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>333</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>195</width>
    <height>320</height>
   </size>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Receiver Options</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="pageDemodVLayout1">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QFormLayout" name="pageDemodFormLayout1">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="horizontalSpacing">
         <number>10</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="typeLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Demod:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>19</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="modeSelector">
          <property name="toolTip">
           <string>Demodulator type (mode)</string>
          </property>
          <property name="statusTip">
           <string>Demodulator type (mode)</string>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <item>
           <property name="text">
            <string>SSB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AM</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FM</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="demodOptions">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="demodSsbOpt">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="sidebandLabel">
            <property name="text">
             <string>Sideband</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sidebandSelector">
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>Lower</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Upper</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="demodAmOpt">
         <widget class="QWidget" name="layoutWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>123</width>
            <height>52</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="pageDemodVlayout2">
           <item>
            <widget class="QCheckBox" name="dcr">
             <property name="toolTip">
              <string>Remove DC offset in audio</string>
             </property>
             <property name="statusTip">
              <string>Remove DC offset in audio</string>
             </property>
             <property name="text">
              <string>DCR</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="syncPll">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Sync PLL</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="demodFmOpt">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QFormLayout" name="pagedemodFormLayout2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="maxdevLabel">
              <property name="text">
               <string>Dev:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="maxdevSelector">
              <property name="toolTip">
               <string>Maximum FM deviation</string>
              </property>
              <property name="statusTip">
               <string>Maximum FM deviation</string>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <item>
               <property name="text">
                <string>Voice (2.5k)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Voice (5k)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>APT (17k)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>BC (75k)</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="emphLabel">
              <property name="text">
               <string>Tau: </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="emphSelector">
              <property name="toolTip">
               <string>Time constant for the FM de-emphasis IIR filter.
For narrow band FM use 530 μs.
50 μs and 75 μs are used for broadcast FM depending on region.
For digital modes it is best to switch it off.</string>
              </property>
              <property name="statusTip">
               <string/>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="currentIndex">
               <number>6</number>
              </property>
              <item>
               <property name="text">
                <string>Off</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>25 μs</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>50 μs</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>75 μs</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>100 μs</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>250 μs</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>530 μs</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1 ms</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="hline2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="pageFilterHLayout1">
      <item>
       <widget class="QLabel" name="nbLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Noise Blanker:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nb1Button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Noise blanker 1</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nb2Button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Noise blanker 2</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="hline3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="filterLabel">
        <property name="text">
         <string>Channel filter presets</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="filterWidthCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Mode dependent filter width</string>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>Wide</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Narrow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>User</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="filterShapeCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Filter shape</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Soft</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sharp</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="hline1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="agcLabel">
        <property name="text">
         <string>AGC</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="agcPresetCombo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>AGC presets</string>
        </property>
        <item>
         <property name="text">
          <string>Fast</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Slow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>User</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QDial" name="bbGainDial">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Baseband gain</string>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDial" name="agcAttackDial">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string>AGC attack time</string>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDial" name="agcHangDial">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string>AGC hang time</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDial" name="agcDecayDial">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string>AGC decay time</string>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="hline5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="sqlLabel">
        <property name="text">
         <string>SQL</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sqlSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Squelch level in dBFS</string>
        </property>
        <property name="minimum">
         <number>-1000</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="pageStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>-1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="hline4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>modeSelector</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>demodOptions</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
